README.md

```diff
@@ -30,8 +30,11 @@ const onLoginButtonClick = async () => {
   const scopes = ["nt:name", "nt:email"];
   const state = window.location.pathname + window.location.search; // or whatever the next URL is
   const url = await nametag.AuthorizeURL(scopes, state);
+  const qrcode = await nametag.AuthorizeURL(scopes, state, true) // 'true' means send QR code
   window.location.assign(url);
 };
+<img alt="Say hellow with Nametag" src={nametag.server + '/button.svg'}/>
```

Comment:
I'm not exactly sure why this is there, but we're using react syntax here, which people might not be familiar with. I don't think it's needed, but if we keep it, perhaps in it's own ```html``` block